third_party/boringssl/src/crypto/refcount_starboard.c

```diff
@@ -18,24 +18,20 @@
 
 #if defined(STARBOARD)
 
-#include "starboard/atomic.h"
-
 void CRYPTO_refcount_inc(CRYPTO_refcount_t *count) {
-  CRYPTO_refcount_t expected = SbAtomicNoBarrier_Load(count);
+  CRYPTO_refcount_t expected = __atomic_load_n(count, __ATOMIC_RELAXED);
```

Comment:
Are you sure we want to go with compiler builtins ( __atomic_load ) rather than C11 standard ones from `<stadomic.h>` ? E.g this : https://en.cppreference.com/w/c/atomic/atomic_load

There's pros and cons to each. Now that we don't have to worry about MSVC, either path should work. 

For starboard toolchains, we added a C11 as requirement as well, so in theory standard way _should_ work, but the builtins are probably more universally available.

Reply:
The C11 atomics are optional and the client is supposed to check __STDC_NO_ATOMICS__ before attempting to use them.  The libcxx code we link on Evergreen already uses the compiler built-ins.

Reply:
Yes, but in practice gcc / clang implement them all the way back to old compilers we use, and compiler builtins _also_ rely on gcc / clang.

See example here: https://godbolt.org/z/jnnzE15a1

The key is to pass `-x c -std=c11` in when compiling with `g++` to ensure C standard is properly set.

Reply:
I thought we wouldn't worry about msvc. Did something change?

Reply:
Updated the change to use c11 as we can get rid of one of the starboard customization files.